### ISSUE TITLE
build issue

### ISSUE THREAD

**Issue by johto** [sfackler/rust-phf#44]
Hi,

I'm trying to build rust-phf on the current version of rustc, and I'm seeing a number of errors like this:

```
src/map.rs:46:9: 53:14 error: failed to resolve. Could not find `iter` in `std`
src/map.rs:46         for (k, v) in self.entries() {
src/map.rs:47             if !first {
src/map.rs:48                 try!(write!(fmt, ", "));
src/map.rs:49             }
src/map.rs:50             try!(write!(fmt, "{:?}: {:?}", k, v));
src/map.rs:51             first = false;
              ...
src/map.rs:46:9: 53:14 error: unresolved name `std::iter::IntoIterator::into_iter`
src/map.rs:46         for (k, v) in self.entries() {
src/map.rs:47             if !first {
src/map.rs:48                 try!(write!(fmt, ", "));
src/map.rs:49             }
src/map.rs:50             try!(write!(fmt, "{:?}: {:?}", k, v));
src/map.rs:51             first = false;
              ...
src/map.rs:46:9: 53:14 error: failed to resolve. Could not find `iter` in `std`
src/map.rs:46         for (k, v) in self.entries() {
src/map.rs:47             if !first {
src/map.rs:48                 try!(write!(fmt, ", "));
src/map.rs:49             }
src/map.rs:50             try!(write!(fmt, "{:?}: {:?}", k, v));
src/map.rs:51             first = false;
              ...
src/map.rs:46:9: 53:14 error: unresolved name `std::iter::Iterator::next`
src/map.rs:46         for (k, v) in self.entries() {
src/map.rs:47             if !first {
src/map.rs:48                 try!(write!(fmt, ", "));
src/map.rs:49             }
src/map.rs:50             try!(write!(fmt, "{:?}: {:?}", k, v));
src/map.rs:51             first = false;
              ...
src/map.rs:46:13: 46:19 error: failed to resolve. Could not find `option` in `std`
src/map.rs:46         for (k, v) in self.entries() {
                          ^~~~~~
src/map.rs:46:13: 46:19 error: unresolved enum variant, struct or const `Some`
src/map.rs:46         for (k, v) in self.entries() {
                          ^~~~~~
src/map.rs:46:9: 53:14 error: failed to resolve. Could not find `option` in `std`
src/map.rs:46         for (k, v) in self.entries() {
src/map.rs:47             if !first {
src/map.rs:48                 try!(write!(fmt, ", "));
src/map.rs:49             }
src/map.rs:50             try!(write!(fmt, "{:?}: {:?}", k, v));
src/map.rs:51             first = false;
              ...
src/map.rs:46:9: 53:14 error: unresolved enum variant, struct or const `None`
src/map.rs:46         for (k, v) in self.entries() {
src/map.rs:47             if !first {
src/map.rs:48                 try!(write!(fmt, ", "));
src/map.rs:49             }
src/map.rs:50             try!(write!(fmt, "{:?}: {:?}", k, v));
src/map.rs:51             first = false;
              ...
```

Is that expected?  I really don't understand the error at all.

**Comment by sfackler** [comment]
Uh, that's weird. What's the output of `rustc --version -v` for you? I can build just this fine on `rustc 1.0.0-nightly (1d00c545e 2015-01-30 19:56:34 +0000)`.

**Comment by johto** [comment]
```
% rustc --version -v
rustc 1.0.0-dev (474b324ed 2015-01-31 03:57:01 +0000)
binary: rustc
commit-hash: 474b324eda10440d6568ef872a7307d38e7de95b
commit-date: 2015-01-31 03:57:01 +0000
host: x86_64-apple-darwin
release: 1.0.0-dev
```

**Comment by sfackler** [comment]
Ok, looks like it was probably due to some change in the way for loops work that landed in the last couple of hours. I'll fix it up, but I need to build rustc first.

**Comment by sfackler** [comment]
Should be fixed now with the `v0.6.1` release.

**Comment by johto** [comment]
Works perfectly.  Thank you!
